canonical-web-and-design/snapcraft.io
# Move progressiveRow to class. Throttle onChange handler

## Done

- Move progressiveRow to standard React component (so isn't initalized and event bound every time the percentage is updated)
- Add [react-throttle-render](https://gitlab.com/ryo33/react-throttle-render) library and selectively throttle components

## Issue / Card

Fixes #2411 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit http://0.0.0.0:8004/snap_name/releases
- Prepare a few progressive releases
- Open the details before you save
- Scroll/ drag the first slider - the others should update smoothly (if use for all checkbox is checked)
